### ISSUE TITLE
h

### ISSUE THREAD

**Issue by IMS212** [ConsistencyPlus/ConsistencyPlus#69]
h

**Comment by IMS212** [comment]
h

**Comment by Ibex80** [comment]
h

**Comment by wafflecoffee** [comment]
2 unhelpful comments were automatically hidden. Show

**Comment by Cart3r1234** [comment]
smh emma ruining the fun

**Comment by IMS212** [comment]
h

**Comment by pjalsGit** [comment]
h

**Comment by identithree** [comment]
h

**Comment by Stereo528** [comment]
h

**Comment by Dragon-Seeker** [comment]
h

**Comment by Redd56** [comment]
h

**Comment by pjalsGit** [comment]
when is this feature coming out?

**Comment by Redd56** [comment]
> when is this feature coming out?

H.h.h on h/h

**Comment by Siuolthepic** [comment]
H

**Comment by Stereo528** [comment]
h

**Comment by Redd56** [comment]
H! HHH??? HH!?!??! HHHHHHhHHhhhHHHhHhhhHHHHhH?!?!??!????!?!?!?!?! HH!!!

**Comment by IMS212** [comment]
h

**Comment by asoji** [comment]
h

**Comment by LizIsTired** [comment]
h

**Comment by StartsMercury** [comment]
h

**Comment by mineland-0066** [comment]
h

**Comment by NickWilde263** [comment]
hhhhhhhhhhhHHHhhhHhhHh....




ello

**Comment by meisme-dev** [comment]
h

**Comment by Stereo528** [comment]
h

**Comment by Dragon-Seeker** [comment]
h

**Comment by maximumpower55** [comment]
h

**Comment by DmanMC** [comment]
h

**Comment by pajicadvance** [comment]
'h' is a good meme because it just doesn't mean anything. It's one of the few letters that really can't be associated with anything. By itself, it just cannot mean anything. The letter "a" is obviously associated with the word, we say it a lot. It's also a letter grade, along with BCDF. There's clear meaning in it. Even a less common letter like X is associated with something, like your "ex" or "exterminate" or something. Maybe even the algebra variable x, along with y and z. They're just too familiar.

But h? It's just h. It literally doesn't rhyme with anything. It doesn't stand for anything. It doesn't mean anything but it still takes effort to pronounce. H is succinct but also not the easiest letter to say in terms of effort. It takes effort to say for how succinct it is, and that makes it even more useless and it's great.

Online, it's a great way to say something without actually saying anything at all. It's the equivalent of just typing "." or just a blank message in a chat, except you're writing something that can be pronounced. It's brilliant.

Close after might come W and E. I'd argue that's why E became a meme before, it's also kind of a standalone letter but there are some differences. I'm sure W has seen its spotlight here and there over the past decade or more.

TL;DR The letter H by itself is so brilliantly useless. That's why it's so good.

**Comment by IMS212** [comment]
h

**Comment by IMS212** [comment]
h

**Comment by Redd56** [comment]
H....

**Comment by asoji** [comment]
h

**Comment by Redd56** [comment]
H?

**Comment by pjalsGit** [comment]
i prefer method h

**Comment by Redd56** [comment]
H!!!!!

**Comment by pjalsGit** [comment]
i don't know.... h....

**Comment by Pythagoras-314** [comment]
### _**h**_

**Comment by asoji** [comment]
h

**Comment by oatmealine** [comment]
h

**Comment by AetherARCNMND** [comment]
h

**Comment by arceuss** [comment]
h

**Comment by ManyBruhMoments** [comment]
fix brocken mod always crash with tlauncher and crystal launcher and make it work because this is your broken mod and i want it workig now

give me error but your issue so
𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫

make it work please please pelawse please pleas epla ea please please please

**Comment by Redd56** [comment]
> fix brocken mod always crash with tlauncher and crystal launcher and make it work because this is your broken mod and i want it workig now
> 
> give me error but your issue so 𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫
> 
> make it work please please pelawse please pleas epla ea please please please

Hh,h h hhh! Hhh?hhhhh-hh, hhh.hhh ?

**Comment by meisme-dev** [comment]
> fix brocken mod always crash with tlauncher and crystal launcher and make it work because this is your broken mod and i want it workig now
> 
> give me error but your issue so 𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫
> 
> make it work please please pelawse please pleas epla ea please please please

tlauncher is deprecated, use reactjs

**Comment by Siuolthepic** [comment]
> fix brocken mod always crash with tlauncher and crystal launcher and make it work because this is your broken mod and i want it workig now
> 
> give me error but your issue so 𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫𒐫
> 
> make it work please please pelawse please pleas epla ea please please please

Fixed in 1.0+1.10.1-Quilt-Depless-alpha.1

**Comment by asoji** [comment]
h

**Comment by maximumpower55** [comment]
h

**Comment by pjalsGit** [comment]
h

**Comment by meisme-dev** [comment]
> h

h

**Comment by Redd56** [comment]
H

**Comment by pjalsGit** [comment]
H??? h... H!!!!!!!!

**Comment by Redd56** [comment]
H h h h h, h h: h. H h: h, h, h, h, h, h.

**Comment by meisme-dev** [comment]
# H

**Comment by IMS212** [comment]
## H

**Comment by pjalsGit** [comment]
![image](https://user-images.githubusercontent.com/25231103/172571176-2f6438fe-d48e-4d99-a5df-eebddb34f34c.png)

**Comment by Redd56** [comment]
H

**Comment by asoji** [comment]
h

![lttstore_water_bottle_but_its_just_h_instead_of_linus_tech_tips](https://user-images.githubusercontent.com/99072163/176549704-4dd90837-e57d-4083-8cfc-b44fa0a63035.jpg)

**Comment by pjalsGit** [comment]
![image](https://user-images.githubusercontent.com/25231103/176638399-1bc563d6-a790-4f45-8565-d7cadf86bbe7.png)

**Comment by meisme-dev** [comment]
# H
## H
### H
#### H
H
##### H
###### H

**Comment by pjalsGit** [comment]
![image](https://user-images.githubusercontent.com/25231103/177525819-a4f34ec8-fc36-43b0-a560-c400ba61ddc1.png)

**Comment by pjalsGit** [comment]
![image](https://user-images.githubusercontent.com/25231103/177525930-469ad840-2a76-466e-ab00-e504012ce66f.png)

**Comment by meisme-dev** [comment]
![image](https://user-images.githubusercontent.com/49573906/177992680-8acbb576-5d58-460a-a765-66715a879d77.png)

**Comment by asoji** [comment]
h

**Comment by MobButcher** [comment]
н

**Comment by pjalsGit** [comment]
![image](https://user-images.githubusercontent.com/25231103/177995749-1bdbcd71-9831-4c28-8593-22425b690287.png)

**Comment by BeansBeefBroccoli** [comment]
![h](https://user-images.githubusercontent.com/69123018/177996079-9e720d30-ecef-46b1-a186-3503e07a90de.png)

**Comment by asoji** [comment]
h?
![h_but_congrats_on_69_comments](https://user-images.githubusercontent.com/99072163/177996102-8b11bf83-2ae7-45aa-a6a3-0fa5f334929f.png)

**Comment by EdgyPotato** [comment]
ℋ

**Comment by BeansBeefBroccoli** [comment]
н

**Comment by pjalsGit** [comment]
![image](https://user-images.githubusercontent.com/25231103/178220441-6a39e200-aeca-4be8-bb9b-9cfc35ee14e8.png)

**Comment by pjalsGit** [comment]
![image](https://user-images.githubusercontent.com/25231103/178732745-35437130-edf7-4ec1-be7d-2f5e2b8b142e.png)

**Comment by Redd56** [comment]
![Screenshot_20220713-230434](https://user-images.githubusercontent.com/12189828/178911501-e7408b96-58b9-44ee-b9ea-445988262b41.png)

**Comment by pjalsGit** [comment]
![image](https://user-images.githubusercontent.com/25231103/178937539-5e3004ba-4f90-4b15-949d-965d9a2a4f39.png)

**Comment by Redd56** [comment]
![Uploading Screenshot_20220714-093145.png…]()

**Comment by Redd56** [comment]
![Screenshot_20220714-093145](https://user-images.githubusercontent.com/12189828/179032359-5bae5548-4cb5-4bbe-88ca-2cdf76f08fc1.png)

**Comment by Cart3r1234** [comment]
This was fun while it lasted, but all great things must come to an end.

_**h**_

**Comment by Pythagoras-314** [comment]
h

**Comment by TropheusJ** [comment]
h

**Comment by Stereo528** [comment]
h
